src/OrleansSQLUtils/Properties/AssemblyInfo.cs

```diff
@@ -21,4 +21,5 @@
 [assembly: InternalsVisibleTo("UnitTests")]
 [assembly: InternalsVisibleTo("TesterInternal")]
 [assembly: InternalsVisibleTo("UnitTestGrains")]
+[assembly: InternalsVisibleTo("Orleans.NonSiloTests")]
```

Comment:
We shouldn't add this. If you are migrating SQL tests, do not migrate them to NonSiloTests because we eventually want them to go to OrleansSqlUtils.Tests (see https://github.com/dotnet/orleans/issues/2011). I'd say either keep them where they are, or start moving to the provider specific test project.